app/Http/Controllers/ApiController.php

```diff
@@ -16,6 +16,10 @@ public static function geocode(Request $request)
             'language' => 'string',
             'referrer' => 'url|required',
             'search' => 'string|required',
+            'north' => 'numeric',
+            'south' => 'numeric',
+            'east' => 'numeric',
+            'west' => 'numeric',
         ]);
```

Comment:
I ran TSML UI locally against this, and noticed when you use the "Search Near Location" feature, this works, but if you refresh the page and it loads, the geocode is made without values for north, south, east and west. Likely timing.

http://localhost:3000/tests/aasanjose?distance=5&search=staples+center&mode=location

Sure it's worth fixing, but also, I feel like if you provide `&north=&south=&east=&west=` it shouldn't fail. Would allow nullable, to treat these as optional:
```
            'north' => 'numeric|nullable',
            'south' => 'numeric|nullable',
            'east' => 'numeric|nullable',
            'west' => 'numeric|nullable',
```

Reply:
good call